docs/contributor-guide/how to/rule.md

```diff
@@ -0,0 +1,226 @@
+# Develop a rule
+
+A `rule` is a check that `sonarwhal` will validate. The API should be
+flexible enough to allow you to implement anything you want easily:
+
+* Validate that all links are `HTTPS`.
+* Integrate with a third party service.
+* Inject JavaScript to execute in the context of the page.
+* etc.
+
+If there is something you want to do and you can’t, or it is not clear
+how to do it, please open an issue.
```

Comment:
Maybe make `open an issue` a link pointing to https://github.com/sonarwhal/sonarwhal/issues/new?

Reply:
OK